assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserCountGraph.js

```diff
@@ -114,98 +52,118 @@ export default function UserCountGraph( { dimensionName, dimensionValue } ) {
 		return <ReportError moduleSlug="analytics" error={ error } />;
 	}
 
-	if ( isZeroReport( report ) ) {
-		return <ReportZero moduleSlug="analytics" />;
-	}
```

Comment:
This is different from before, but good point on removing the `isZeroReport` check from `UserCountGraph` and `TotalUserCount`. These might be 0 because of a certain filtered dimension having 0 users. In that case, it still makes sense to show the 0s instead of the `ReportZero` box since the latter only exists to indicate that likely your site doesn't have enough data for meaningful results. For these two components though, if they are rendered at all, it means there _is_ data overall, so 👍 

Nice UX improvement here (or we could even argue the behavior from before was a bug).